### ISSUE TITLE
"TypeError: pop() takes no arguments (1 given)" occurs when using Pager generator

### ISSUE THREAD

**Issue by Talvalin** [tableau/server-client-python#133]
When trying to use the [documented page options](https://tableau.github.io/server-client-python/docs/page-through-results#set-pagination-options), the following error occurs

```
all_users = list(TSC.Pager(server.users, request_options))
  File "<python_install_path>\lib\site-packages\tableauserverclient\server\pager.py", line 30, in __iter__
    yield current_item_list.pop(0)
TypeError: pop() takes no arguments (1 given)
```

Script example:
```
request_options = TSC.RequestOptions(pagesize=1000)
all_users = list(TSC.Pager(server.users, request_options))
```

**Comment by t8y8** [comment]
This was already fixed in #109 -- are you on an old version?

**Comment by Talvalin** [comment]
Fail. I had 0.2 installed, which updated to 0.3 when I ran `pip install --upgrade`. Guess this can be closed out then?
